Assignment2Q1.java

```diff
@@ -0,0 +1,117 @@
+import java.util.LinkedList;
+import java.util.List;
+
+public class Assignment2Q1 {
+	public static void main(String[] args) {
+		BinaryTreeNode<Integer> n1 = new BinaryTreeNode<>(1, null, null);

```

Comment:
But I use those links to create the tree. If I don't link the nodes, I won't have a binary tree.